packages/mint-components/src/components/sqm-task-card/progress-bar/progress-bar.feature

```diff
@@ -0,0 +1,73 @@
+@author:
+@owner:
+Feature: Task Card Progress Bar
+
+	Scenario Outline: Progress Bar
```

Comment:
The scenario titles in this file need to be more descriptive of the steps that they are speccing out